docs/vscode-online.md

```diff
@@ -0,0 +1,31 @@
+# VSCode online
+
+It is possible to host an instance of Visual Studio Code in the cloud and have an URL with all your files (protectd with a password), being able to use it in any browser and device.
+
+## Foam with DigitalOcean
+
+- Requirements: a DigitalOcean account and Foam hosted in git (GitHub, Azure DevOps, etc). Extra: Needed a domain and a certificate to have a url like: https://foam.yourdomain.com/
+
+### Create the droplet
+- Go to: Droplets, push button: "Create Droplet" and over Choose and image go to marketplace and select: "code-server"
```

Comment:
I am not sure I understand this sequence, can you plz rephrase it or break it down further?

Reply:
I tried to explain it better. Please let me know if it's ok for you.

Reply:
The updates on the rest of the guide are really good!
I am still a bit confused about this specific line :) what do you mean by "over Choose and image"? Is it `hover "Choose an image"`? then you go to a "marketplace" label/button?
Maybe you can add quotes around labels to highlight them from the instruction

Reply:
It'd be an instead and !! 😂
Fixing it now.